footil/xiter.py

```diff
@@ -0,0 +1,23 @@
+"""Some iteration helpers."""
+
+
+def iterate_limit(iterable, limit=-1):
+    """Iterate items up to specified limit.
+
+    If limit is greater than items in iterable, it will stop gracefully.
+
+    Args:
+        iterable: to iterate safely.
+        limit: how many items to iterate. -1 means all.
```

Comment:
```suggestion
        limit: how many items to iterate. -1 means all (default: {-1}).
```

Reply:
Its written as `({default: -1})`.

Reply:
Or not. Seems in some cases, its used incorrectly:)